dev/tests/unit/Magento/FunctionalTestFramework/Composer/ComposerInstallTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace tests\unit\Magento\FunctionalTestFramework\Test\Util;
+
+use Magento\FunctionalTestingFramework\Composer\ComposerInstall;
+use Magento\FunctionalTestingFramework\Util\MagentoTestCase;
+
+class ComposerInstallTest extends MagentoTestCase
+{
+    /**
+     * ComposerInstall instance to be tested
+     *
+     * @var ComposerInstall
+     */
+    private static $composer;
```

Comment:
Let's change this to just be `private $composer;` and change `setUpBeforeClass()` to just `setUp()`.

Since this is initialized only once in the test, the object's state is subject to change due to test's manipulation, which set a bad pattern if we ever do any state tracking in that class in the future.

Also being the class under test, we should really initialize once per test.

Reply:
Fixed